src/js/import.js

```diff
@@ -241,6 +233,31 @@
     });
   }
 
+  function addClickableButtonEventListener(button, CSVHeaders){
+    button.click(function() {
+      if(button.is(":checked")){
+        var potentialComplaint = button.parent().nextAll();
+        potentialComplaint.remove();
+        var className = button.attr('class').split(" ");
+        var serpType = className[className.length-1];
+        button.parent().parent().append(elCheckboxAndSelect(serpType, true));
+        var newSelect = button.parent().next().find(".import-select");
+        addOptions(CSVHeaders, newSelect);
+        var newCheckbox = button.parent().next().find(".import-checkbox");
+        addClickableButtonEventListener(newCheckbox, CSVHeaders);
+      } else {
+        button.parent().nextAll().remove();
```

Comment:
I made these "one-time used" vars because I thought it made the code more readable and also so that the lines wouldn't be so long. They cannot be made before the "if" since they are created by the `button.parent().parent().append(elCheckboxAndSelect(serpType, true));` line just above what you see here. 

But if you still want me to change it let me know.